low-level/actuators/impl/generic/node_hw.py

```diff
@@ -367,3 +363,11 @@ def _response_to_dict(self, data, split_char=':', dict_keys=None, many_sensors=F
             return many_sensors_data
 
         return properties
+
+    def _read_config(self):
+        """ Read Configurations from global config"""
+        conf_list = ['site_id', 'node_id', 'rack_id']
+        response_dict = self._global_conf.fetch_global_config(conf_list)
```

Comment:
this is infact increasing the code compared to previously.

Why cant we have like
loc  =  self._global_conf.fetch_global_config(['site_id', 'node_id', 'rack_id'])

and use loc['site_id'] directly where ever needed. Anyways it would have default have if error occurs while fetching an config.